src/Codec/Xlsx/Types/Common.hs

```diff
@@ -385,3 +411,76 @@ instance ToAttrVal ErrorType where
   toAttrVal ErrorNum = "#NUM!"
   toAttrVal ErrorRef = "#REF!"
   toAttrVal ErrorValue = "#VALUE!"
+
+type RowIndex = Int
+type ColIndex = Int
+
+#ifdef USE_MICROLENS
```

Comment:
Since micro-lens denies the existence of prisms, I pasted the splice that's generated from makePrisms, then I copied over the definitions from control.lens for the `prism` function as well. Essentially this is doing the template haskell by hand.